test/vc-data-model-1.0/60-zkp.js

```diff
@@ -128,28 +78,11 @@ describe('Zero-Knowledge Proofs (optional)', function() {
       expect(hasType(doc, 'VerifiablePresentation')).to.be.true;
     });
 
-    describe('Each verifiable credential...', function() {
-      it('MUST have a `credentialSchema` member', async function() {
-        const doc = await util.generatePresentation('example-015-zkp-vp.jsonld', generatorOptions);
-        if (Array.isArray(doc.verifiableCredential)) {
-          let creds = [].concat(doc.verifiableCredential)
-          for(let cred of creds){
-            const isArray = Array.isArray(cred.credentialSchema) && cred.credentialSchema.length > 0;
-            const isObject = cred.credentialSchema && typeof cred.credentialSchema.id === 'string';
-            expect(isArray || isObject).to.be.true;
-          }
-        } else {
-          // only one credential
-          const isArray = Array.isArray(doc.verifiableCredential.credentialSchema) && doc.verifiableCredential.credentialSchema.length > 0;
-          const isObject = doc.verifiableCredential.credentialSchema && typeof doc.verifiableCredential.credentialSchema.id === 'string';
-          expect(isArray || isObject).to.be.true;
-        }
-      });
-      it('MUST contain a credentialSchema (negative - credentialSchema missing)', async function() {
-        await expect(util.generate(
-          'example-015-zkp-vp-bad-no-credential-schema.jsonld', generatorOptions))
-          .to.be.rejectedWith(Error);
-      });
+    it.skip('credential definition MUST be defined in credentialSchema', async function() {
+    /*If a credential definition is being used, the credential
+    definition MUST be defined in the credentialSchema property,
+    so that it can be used by all parties to perform various cryptographic
```

Comment:
```suggestion
    so all parties can use it to perform various cryptographic
```

Reply:
@clehner -- If https://github.com/w3c/vc-test-suite/pull/127#issuecomment-1034242871 is asking me about the above rephrasing, then I would definitely change it here, and if such editorial change can still be made in the other doc, then I would change it there as well.